surveys/templates/survey_submitted_detail.html

```diff
@@ -112,6 +112,18 @@ <h3 class="card-title">Analyze</h3>
     });
   }
 
+  function toggleMoveButtonVisibility() {
+    // Check to see if there is only one chart on the page, and if there is, hide
+    // the move buttons on that chart
+    if ($(chartContainer).filter(':visible').length === 1) {
```

Comment:
Could we also check here if a single chart's position is first or last and hide the up and down arrows, respectively?

Reply:
Great point! Added.